components/qma6100p/test/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+idf_component_register(SRCS "qma6100p_test.c"
```

Comment:
All new components' tests should be a standalone test_apps. eg. like here https://github.com/espressif/esp-bsp/tree/master/components/lcd/esp_lcd_gc9503/test_apps

This way the test application for QMA6110P is self-contained application, which make it more scalable